src/CMakeLists.txt

```diff
@@ -118,6 +118,40 @@ CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/core/deviceExecutionHandlers.cpp.in $
 forte_replacefile_if_changed(${CMAKE_BINARY_DIR}/core/deviceExecutionHandlers_new.cpp ${CMAKE_BINARY_DIR}/core/deviceExecutionHandlers.cpp)
 file(REMOVE ${CMAKE_BINARY_DIR}/core/deviceExecutionHandlers_new.cpp)
 
+#############################################################################
+# Configure Timers
+#############################################################################
+
+GET_PROPERTY(TIMERHANDLER_CLASS GLOBAL PROPERTY FORTE_TIMERHANDLER_CLASS)
+GET_PROPERTY(TIMERHANDLER_FILENAME GLOBAL PROPERTY FORTE_TIMERHANDLER_FILENAME)
+GET_PROPERTY(TIMERHANDLER_NAME GLOBAL PROPERTY FORTE_TIMERHANDLER_NAME)
+GET_PROPERTY(TIMERHANDLER_DEFAULT GLOBAL PROPERTY FORTE_TIMERHANDLER_DEFAULT)
+
+if(NOT DEFINED TIMERHANDLER_DEFAULT)
+  set(TIMERHANDLER_DEFAULT "\"\"")
+endif()
+
+SET(TIMERHANDLER_INCLUDE "")
+SET(TIMERHANDLER_CREATE "")
+
+LIST(LENGTH TIMERHANDLER_CLASS TIMERHANDLER_CLASS_LEN)
+math(EXPR TIMERHANDLER_CLASS_LEN ${TIMERHANDLER_CLASS_LEN}-1)
+
+FOREACH(POS RANGE ${TIMERHANDLER_CLASS_LEN})
```

Comment:
as there can only be one timerhandler, why not just generating the creation function for this one. This avoids the need for the string param of the factory and would make it clearer what is going on. Also the code would only have the timerhandler needed.

Reply:
During some trace testing, I need to be able to change the timer at runtime. I run the tests with the regular timer and then switch to the `fake_timer` to simulate different things.

Reply:
How would I change the timer handler during runtime?

Just by calling `CTimerHandlerFactory::setTimeHandlerNameToCreate` and `CTimerHandlerFactory::createTimerHandler`?

Reply:
> How would I change the timer handler during runtime?
> 
> Just by calling `CTimerHandlerFactory::setTimeHandlerNameToCreate` and `CTimerHandlerFactory::createTimerHandler`?

Yes, that's the way of doing it. My only use case for now is to change it in the unit test. 
1. I created a forte device with the default timer
2. execute some FBs 
3. get some result. 

Then  change the timer and execute the same three steps as above. 

Adding a command line could be an option for later if needed. I was also thinking if this might be needed in the future so it can be changed from 4diac-IDE with some custom command.